src/staverepetition.ts

```diff
@@ -101,95 +99,52 @@ export class Repetition extends StaveModifier {
   }
 
   drawCodaFixed(stave: Stave, x: number): this {
-    const y = stave.getYForTopText(stave.getNumLines()) + this.yShift;
-    Glyph.renderGlyph(
-      stave.checkContext(),
-      this.x + x + this.xShift,
-      y + Tables.currentMusicFont().lookupMetric('staveRepetition.coda.offsetY'),
-      40,
-      'coda',
-      { category: 'coda' }
-    );
+    const y = stave.getYForTopText(stave.getNumLines());
+    this.text = '\ue048' /*coda*/;
+    this.renderText(stave.checkContext(), x, y + Tables.lookupMetric('Repetition.coda.offsetY'));
     return this;
   }
 
-  drawSignoFixed(stave: Stave, x: number): this {
-    const y = stave.getYForTopText(stave.getNumLines()) + this.yShift;
-    Glyph.renderGlyph(
-      stave.checkContext(),
-      this.x + x + this.xShift,
-      y + Tables.currentMusicFont().lookupMetric('staveRepetition.segno.offsetY'),
-      30,
-      'segno',
-      { category: 'segno' }
-    );
+  drawSegnoFixed(stave: Stave, x: number): this {
+    const y = stave.getYForTopText(stave.getNumLines());
+    this.text = '\ue047' /*segno*/;
+    this.renderText(stave.checkContext(), x, y + Tables.lookupMetric('Repetition.segno.offsetY'));
     return this;
   }
 
   drawSymbolText(stave: Stave, x: number, text: string, drawCoda: boolean): this {
     const ctx = stave.checkContext();
-
-    ctx.save();
-    ctx.setFont(this.textFont);
-
     let textX = 0;
-    let symbolX = 0;
-    const modifierWidth = stave.getNoteStartX() - this.x;
+
+    this.text = text;
+    if (drawCoda) {
+      this.text += ' \ue048' /*coda*/;
```

Comment:
Is this leading space intentional? It looks unexpected. If it is intentional, then please document the reason why with a comment here.

Reply:
Is this because this method is for drawing D.S. al ⊕ so you need the space before the coda string? 

Can you add that as a comment here? I'll go ahead and merge, so you can add it to your next PR.